styles.css

```diff
@@ -0,0 +1,180 @@
+@import url('https://fonts.googleapis.com/css2?family=Lato:wght@300&display=swap');
+@import url('https://fonts.googleapis.com/css2?family=Open+Sans:wght@300&display=swap');
+
+
+*{
+    margin: 0;
+    padding: 0;
+    box-sizing: border-box;
+}
+
+:root{
+    --default-color : #172B4D;
+    --primary-color : #50A8EA;
+    --secondary-color : #9195A0;
+
+    --bg-color: #FAFAFA;
+
+    --box-shadow: 0 0 10px #172a3a33;
+    --box-shadow-color: #172a3a33;
+
+    --danger-color : #D82148;
+    --success-color : #019267;
+    --warning-color : #F58840;
+
+    --primary-hover : #172B4D;
+    --secondary-hover: #D1D1D1;
+    --danger-hover : #FF7878 ;
+    --success-hover: #008E89;
+
+    --active-hover : #084594;
+
+    --off-white : #F6F9FC;
+    --active-color : #1572A1;
+
+    --logo-fontsize: 1.4rem;
+    --nav-item-fontsize: 1rem;
+
```

Comment:
I saw most of the same color variables in component.css. You could look into this.